tests/network/stuntime/test_stuntime_measurement.py

```
@@ -0,0 +1,75 @@
+        Preconditions:
+            - Running Fedora migrated-VM on OVN localnet secondary network, running on worker1.
+            - Running Fedora static-VM on OVN localnet secondary network, running on worker1.
+            - Ping running at 100 ms intervals (ping -D -O -i 0.1) from initiator VM to peer.
+
+        Steps:
+            1. Initiate live migration of the migrated VM along the specified path.
+            2. Parse ping output for connectivity gap (last success before loss to first success after recovery).
+            3. Compare measured stuntime against per-scenario threshold.
+
+        Expected:
+            - Measured stuntime does not exceed the scenario threshold (min(max×4, 5s)).
```

Ruff (0.15.2)

[warning] Docstring contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF002)